uber/templates/jobs/staffers.html

```diff
@@ -76,4 +98,48 @@
     [L]: <i>Staffers marked with a "L" are working load-in and/or load-out</i>
 </div>
 
+
+{% if c.HAS_ACCOUNTS_ACCESS %}
+<script src="../static/js/jquery.form.js"></script>
+<script>
+var list_of_ids = [];
+
+$("#admin_settings").ajaxForm(function(message){
+    toastr.info(message);
+    setTimeout(function(){
+        location.reload(true);
+    }, 1500);
+});
+
+var checkAll = function(){
+    $.each( $("input[name^='id']"), function () {
+      $(this).prop("checked", true);
+    });
+};
+
+var uncheckAll = function(){
+    $.each( $("input[name^='id']"), function () {
+      $(this).prop("checked", false);
+    });
+};
+
+var adminList = function(id){
+    var index = list_of_ids.indexOf(id);
+    if(index == -1){
+        list_of_ids.push(id);
+    }
+    else {
+        list_of_ids.splice(index, 1);
+    }
+    $("#count").text(list_of_ids.length);
+};
+
+var add_admin_list = function(){
```

Comment:
See comment on line 78. I suggest you instead do something like (untested syntax):

```
$("#admin_settings").submit( function(eventObj) {
        $('#ids').attr('value', function(i, val) {
            return  $('.checkboxClass:checked').map(function() {
                 return this.id;
           };
        });
        return true;
});
```